packages/node/src/behaviors/door-lock/DoorLockServer.ts

```diff
@@ -5,13 +5,22 @@
  */
 
 import { DoorLock } from "#clusters/door-lock";
+import { ImplementationError } from "#general";
 import { DoorLockBehavior } from "./DoorLockBehavior.js";
 import LockState = DoorLock.LockState;
 
 /**
  * This is the default server implementation of {@link DoorLockBehavior}.
  */
 export class DoorLockServer extends DoorLockBehavior {
+    override initialize() {
+        if (!this.state.supportedOperatingModes.alwaysSet) {
```

Comment:
This doesn't test all of the bits (=== 2047 might be better?) and only catches values at init time. Nbd I suppose given that this is something of a hack anyway

Reply:
adjusted,.,and yes if someone wants to hack it that he always can ... we can just try to support :-)